src/components/nav-bar/nav-bar.tsx

```diff
@@ -36,7 +36,7 @@ export const NavBar: React.FunctionComponent<NavBarProps> = (props: NavBarProps,
                 {(props.items || []).map((item) => (
                     <Tooltip content={item.title} placement='right' arrow={true} key={item.path + item.title}>
                         <div className={classNames('nav-bar__item', { active: isActiveRoute(locationPath, item.path) })}
-                            onClick={() => context.router.history.push(item.path)}>
+                            onClick={() => context.router.history.push(item.path)} role="checkbox">
```

Comment:
> * At the moment, the navItems do not show up as links, because they do not contain a typical `<a href>`

why not use an actual `<Link>` component here? Same as https://github.com/argoproj/argo-ui/pull/222#discussion_r1597553119, that would resolve #166 as well

Reply:
I tried a few variations with a `<Link>` component but I don't know how to make this work. (I have little experience with web development). However, I think that it is a Tooltip-thing. If you look at other Tooltip examples, they have the same issue. Like here, https://react-tooltip.com/docs/getting-started, the Tooltip buttons are clickable but do not show up as real links.

![tool-tip-demo](https://github.com/user-attachments/assets/f3f93dfe-82d7-4117-ac10-b397ac6bd987)